core/src/main/scala/com/stratio/crossdata/launcher/SparkJobLauncher.scala

```diff
@@ -37,8 +38,9 @@ object SparkJobLauncher extends SparkLoggerComponent with CrossdataSerializer {
   def getSparkStreamingJob(crossdataConfig: Config, streamingCatalog: XDStreamingCatalog, ephemeralTableName: String)
                           (implicit executionContext: ExecutionContext): Try[SparkJob] = Try {
     val streamingConfig = crossdataConfig.getConfig(StreamingConfPath)
+    val launcherConfig=crossdataConfig.getConfig(LauncherKey)
     val sparkHome =
-      Properties.envOrNone("SPARK_HOME").orElse(Try(streamingConfig.getString(SparkHomeKey)).toOption).getOrElse(
+      Properties.envOrNone("SPARK_HOME").orElse(Try(launcherConfig.getString(SparkHomeKey)).toOption).getOrElse(
```

Comment:
What about this? 

`launcherConfig.getString(SparkHomeKey) orElse streamingConfig.getString(SparkHomeKey)`

This would let to define the spark home in both sections (streaming and launcher) of the configuration